### PR TITLE
fix: excluding hilla for a gradle project

### PR DESCRIPTION
**Pull request by caalador**
documents vaadin/platform#6519

### CODE REVIEW

**Review by mcollovati — APPROVED** [review]

File: articles/upgrading/index.adoc [hunk 1]
```diff
@@ -392,9 +390,9 @@ When using maven:
 </dependency>
 ----
 
-When using gradle:
+This example is for when you're using Gradle:
 
-.build.gradle
+.Build Gradle
```
Comment (mcollovati) [inline articles/upgrading/index.adoc:395]: `build.gradle` is the name of the file. It seems a bit incorrect to me to change it to `Build Gradle`.
Same for `gradle.properties`.

Comment (caalador) [reply articles/upgrading/index.adoc:395]: does our template support `.[filename]build.gradle` else I guess we should add build.gradle and gradle.properties to the ignore list like we have for pom.xml

**Comment by mshabarov** [comment]
@caalador could you please rebase to `latest`, as now the PR has conflicts.

File: articles/upgrading/index.adoc (hunk 1 above)
Comment (jouni) [reply articles/upgrading/index.adoc]: For code example titles, you can do ``.`build.gradle` ``, or if you really want to, ``.[filename]`build.gradle` ``

In any case, monospace formatted text (backticks) is excluded from Vale rules.

**Pull request merged**